lib/stack/index.js

```diff
@@ -159,12 +159,30 @@ export function Stack (http, data) {
      *
      * client.stack({ api_key: 'api_key'}).globalField('globalField_uid').fetch()
      * .then((globalField) => console.log(globalField))
+     * 
+     * client.stack({ api_key: 'api_key'}).globalField('globalField_uid', { api_version: '3.2' }).fetch()
+     * .then((globalField) => console.log(globalField))
+     * 
+     * client.stack({ api_key: 'api_key'}).globalField({ api_version: '3.2' }).fetch()
+     * .then((globalField) => console.log(globalField))
```

Comment:
@sunil-lakshman , We cannot call the fetch method without the uid. Right? I think the second example can be removed.